flutter-studio/src/io/flutter/project/FlutterProjectStep.java

```diff
@@ -193,12 +193,9 @@ private void updateSdkField(JTextComponent sdkEditor) {
     if (moduleName.isEmpty()) {
       return errorResult("Please enter a name for the " + getContainerName() + ".");
     }
-    //noinspection ConstantConditions
-    if (getModel().projectType().get().get() == FlutterProjectType.APP) {
-      File loc = new File(FileUtil.toSystemDependentName(getModel().projectLocation().get()), getModel().projectName().get());
-      if (loc.exists()) {
-        return errorResult("Project location already exists: " + loc.getPath());
-      }
+    File loc = new File(FileUtil.toSystemDependentName(getModel().projectLocation().get()), getModel().projectName().get());
```

Comment:
`loc` ==> `location`?

Reply:
Yeah, 'location' could be spelled out but I didn't actually edit those lines in this PR. (The indentation changed due to reformatting.)